packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -57,60 +57,65 @@ describe('background sync queue test', () => {
     await backgroundSyncQueue.cleanupQueue();
   });
 
-  it('check defaults', () => {
+  it('should get default values if nothing is given in constructor', () => {
```

Comment:
I see how that is making everything simpler, but that might block the fixes/ feature request for a longer time, can we skip this part and do it gradually instead?